tests/integration_tests/framework/docker.py

```diff
@@ -225,7 +223,7 @@ def _set_container_id_and_ip(container_details):
 
 def _retry(func, exceptions, cleanup=None):
     err = None
-    for _ in range(2600):
+    for _ in range(1200):
```

Comment:
I know it's unrelated to the change you're making here, but is it worth migrating to the retrying library?

Reply:
Maybe. I don't really want to add a dependency for this one use. Also I feel that retrying, with its arcane argument names, is way less readable than just writing it out (note that we are doing non-basic stuff like a cleanup handler)

Anyway, now I noted that this function isn't even used anymore. Now, just the wait-for-starter is used, with its retries and timeouts. So I removed this function instead.

Reply:
<thumbs up>